modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/components/ShareFormModal/ShareFormModal.es.js

```diff
@@ -51,7 +48,17 @@ class ShareFormModal extends Component {
 		}
 
 		super.disposeInternal();
-		this._eventHandler.removeAllListeners();
+
+		const shareURLButton = document.querySelector(
+			'.nav-item > .lfr-ddm-share-url-button'
+		);
+
+		if (shareURLButton) {
+			shareURLButton.removeEventListener(
+				'click',
+				this._handleShareButtonClicked
```

Comment:
Another function `!==` to `bind`-ed version of the same, so I don't think this works.